types/relay-runtime/lib/util/JSResourceReference.d.ts

```diff
@@ -0,0 +1,5 @@
+export interface JSResourceReference<TModule> {
```

Comment:
do we already have this in react-relay? Should we consume from there, or share this one over to that one?

Reply:
Ah yep, we do. Will import that one.

Reply:
my gut tells me relay-runtime should own this type, imported by the react one. 

So leave this here, we can marry up the other once we good.